activate

```diff
@@ -72,15 +76,16 @@ serial="${serial%"${serial##*[![:space:]]}"}"
 live=false
 
 # report in
-if ! out=$(wget -O - -q "$SCRIPT?image=$image&release=$release&vendor=$vendor&product=$product&serial=$serial&live=$live"); then
-    echo "wget failed"
+URL="${SCRIPT}?image=${image}&release=${release}&vendor=${vendor}&product=${product}&serial=${serial}&live=${live}"
```

Comment:
Regarding your comment about needing `jq`, we can do the PUT with a templated string that ends up as valid json. If you want I can do that when I get a chance.